internal/oci/container.go

```diff
@@ -55,33 +56,34 @@ type Container struct {
 	// this is the /var/run/storage/... directory, erased on reboot
 	bundlePath string
 	// this is the /var/lib/storage/... directory
-	dir                string
-	stopSignal         string
-	imageName          string
-	imageID            *storage.StorageImageID // nil for infra containers.
-	mountPoint         string
-	seccompProfilePath string
-	conmonCgroupfsPath string
-	crioAnnotations    fields.Set
-	state              *ContainerState
-	opLock             sync.RWMutex
-	spec               *specs.Spec
-	idMappings         *idtools.IDMappings
-	terminal           bool
-	stdin              bool
-	stdinOnce          bool
-	created            bool
-	spoofed            bool
-	stopping           bool
-	stopLock           sync.Mutex
-	stopTimeoutChan    chan int64
-	stopWatchers       []chan struct{}
-	pidns              nsmgr.Namespace
-	restore            bool
-	restoreArchive     string
-	restoreIsOCIImage  bool
-	resources          *types.ContainerResources
-	runtimePath        string // runtime path for a given platform
+	dir        string
+	stopSignal string
+	// Some name of the image imageID, if any; it may have NO RELATIONSHIP to the users’ requested image name.
+	someNameOfTheContainersImage *references.RegistryImageReference
```

Comment:
personally, I think continuing to refer to it as `imageName` and commenting the caveats is sufficient.

Reply:
Reverted (but kept the explicit type).